refinery/core/utils.py

```diff
@@ -21,6 +21,9 @@
 import core
 import data_set_manager
 
+from core.search_indexes import DataSetIndex
+from data_set_manager.search_indexes import NodeIndex
```

Comment:
@hackdna For example we were referencing the indexes before according to our style guide, but things were broken:
```
(refinery-platform)vagrant@refinery:/vagrant/refinery$ python
Python 2.7.6 (default, Oct 26 2016, 20:30:19)
[GCC 4.8.4] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import core
>>> core.search_indexes.DataSetindex
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'search_indexes'
>>> import data_set_manager
>>> data_set_manager.search_indexes.NodeIndex
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'search_indexes'
```